nemo/collections/common/tokenizers/en_ja_tokenizers.py

```diff
@@ -25,28 +25,12 @@ class EnJaProcessor:
 
     def __init__(self, lang_id: str):
         self.lang_id = lang_id
-        self.moses_tokenizer = MosesTokenizer(lang=lang_id)
-        self.moses_detokenizer = MosesDetokenizer(lang=lang_id)
-        self.normalizer = MosesPunctNormalizer(
-            lang=lang_id, pre_replace_unicode_punct=True, post_remove_control_chars=True
-        )
 
     def detokenize(self, tokens: List[str]) -> str:
-        """
-        Detokenizes a list of tokens
-        Args:
-            tokens: list of strings as tokens
-        Returns:
-            detokenized Japanese or English string
-        """
-        return self.moses_detokenizer.detokenize(tokens)
+        return ' '.join(tokens)
 
     def tokenize(self, text) -> str:
-        """
-        Tokenizes text using Moses. Returns a string of tokens.
-        """
-        tokens = self.moses_tokenizer.tokenize(text)
-        return ' '.join(tokens)
+        return text
```

Comment:
We should revert these changes?

Reply:
I think it's fine to keep this as is if all of the latest En/Ja models don't use any moses tokenization/normalization.